src/main/docs/guide/ioc/lifecycle.adoc

```diff
@@ -10,9 +10,9 @@ snippet::io.micronaut.docs.lifecycle.V8Engine[tags="imports, class", indent=0]
 
 To manage when a bean is constructed, see the section on <<scopes, bean scopes>>.
 
-== When The Context Closes
+== When The Bean Is Destroyed
 
-To invoke a method when the context is closed, use the `javax.annotation.PreDestroy` annotation:
+To invoke a method when the bean is destroyed, use the `javax.annotation.PreDestroy` annotation:
```

Comment:
should not we encourage to use `jakarta.annotation.PreDestroy` instead `javax.annotation.PreDestroy`?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/7262/commits/b580bfcbda8ce21a05bf0382a945177a50dfdb81